website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "ibm"
+page_title: "IBM : docs-ibm-resource-is-virtual-endpoint-gateway-ip"
+sidebar_current: "ibm_is_virtual_endpoint_gateway_ip"
+description: |-
+  Manages IBM Virtual endpoint gateway IP
+---
+
+# ibm\_is_virtual_endpoint_gateway
+
+Provides a Virtual endpoint gateway  resource. This allows Virtual endpoint gateway IP to be created, updated, and cancelled.
+
+
+## Example Usage
+
+In the following example, you can create a VPN gateway IP:
+
+```hcl
+resource "ibm_is_virtual_endpoint_gateway_ip" "virtual_endpoint_gateway_ip" {
+	gateway_id = ibm_is_virtual_endpoint_gateway.endpoint_gateway.id
+	reserved_ip_id = "0737-5ab3c18e-6f6c-4a69-8f48-20e3456647b5"
+}
+
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `gateway_id` - (Required, string,ForceNew) Endpoint gateway ID
+* `reserved_ip_id` - (Required, string,ForceNew) Endpoint gateway IP id
+
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of the VPN gateway connection. The id is composed of \<gateway_id\>/\<gateway_ip_id\>.
+* `name` - Endpoint gateway IP name
+* `created_at` -  Endpoint gateway IP created date and time
+* `resource_type` -  Endpoint gateway IP resource type
```

Comment:
It gives "address" also one of the computed

Reply:
handled